PCLConfig.cmake.in

```diff
@@ -619,7 +619,7 @@ foreach(component ${PCL_TO_FIND_COMPONENTS})
             INTERFACE_INCLUDE_DIRECTORIES "${PCL_${COMPONENT}_INCLUDE_DIRS}"
             INTERFACE_LINK_LIBRARIES "${PCL_${COMPONENT}_LINK_LIBRARIES}"
         )
-      else()
+      else(CMAKE_VERSION VERSION_LESS 3.11)
```

Comment:
Shouldn't this be `elseif`?

Reply:
Yes, thanks for noticing and sorry for the noise, fixed in https://github.com/PointCloudLibrary/pcl/pull/3341/commits/0801d261af61e71d90e44d9fb27a63584d5abe28, let me know if the rest of the PR seems ok so I can rebase it in the parent commit.